backdoor-ctf-2015/binary/echo/README.md

```diff
@@ -17,4 +17,4 @@
 
 ## Other write-ups and resources
 
-* none yet
\ No newline at end of file
+* https://shankaraman.wordpress.com/2015/04/02/backdoor-ctf-2015-echo-writeup/
```

Comment:
Sorry for the inconvenience. Will make it as per your advice next time.

Thank you for notifying.


Regards,
Shankarraman.

Blog <http://shankaraman.wordpress.com/> | Linkedin
<https://www.linkedin.com/pub/shankar-raman/78/69b/b4> | Twitter
<https://twitter.com/@h1dd3ntru7h> | G+
<https://plus.google.com/+ShankarRaman/posts>